uwblueprint/focus-on-nature
# feat: view camper details modal

## Notion ticket link

[Camp Overview: Show registered camper information](https://www.notion.so/uwblueprintexecs/Task-Board-F22-9e03bf9f7c4343f09a8a5c16dc48a12a?p=56ad65fe7dc64a96a3c17155fa737b52&pm=c)



## Implementation description
* Added a new modal, called the `ViewCamperModal` to list camper details



## Steps to test
1. Navigate to http://localhost:3000/admin/camp/63139c7bc3d7b55b44a01531 (Guelph camp), click on the three dots of a camper row, and click view details
2. Scroll through the modal, and make sure that opening and closing functionality works
![image](https://user-images.githubusercontent.com/30396273/193480757-b2825770-d09c-4a41-99bf-0ae63f706df3.png)
![image](https://user-images.githubusercontent.com/30396273/193480770-47e4ca24-e3bd-4c80-881d-a6b641a26f36.png)
![image](https://user-images.githubusercontent.com/30396273/193481394-9ab31de8-bb09-4552-b8fe-08224020fb40.png)



## What should reviewers focus on?
* Under Camper Information in the modal, the text under allergies and additional assistance are now disabled textareas, let me know how this looks (deviates from the design)
* General code logic, notably the drop off and pickup time table implementation (implemented in PickupDropoffTableRow.tsx)
* Review the empty cases of no allergies or no additional needs or no form responses (third picture)


## Checklist
